templates/navigation.tmpl

```diff
@@ -0,0 +1,9 @@
+{{ define "navigation" }}
+<ul>
+  {{ if ne .Title "home" }}<li><a href="/">Home</a></li>{{ end }}
```

Comment:
What is `ne` supposed to be?

Reply:
Not Equal. See: https://golang.org/pkg/text/template/#hdr-Functions